src/lib.js

```diff
@@ -748,10 +777,19 @@ var program,
             bodyArr.push(this.trimQuotesFromString(dataUrlencode));
             bodyArr.push(this.trimQuotesFromString(dataAsciiString));
 
-            request.body.mode = 'raw';
-            request.body.raw = _.join(_.reject(bodyArr, (ele) => {
-              return !ele;
-            }), '&');
+            const rawDataString = _.join(_.reject(bodyArr, (ele) => {
+                return !ele;
+              }), '&'),
+              isGraphqlBody = this.checkIfGraphql(rawDataString);
```

Comment:
`isGraphqlBody` variable naming suggests the value to be of type `boolean`. Which is not the case here as it's an `object`. Can we come up with a better name?

Reply:
Changed `isGraphqlBody` to `graphqlRequestData`.